### ISSUE TITLE
Change in representation of object in 3.9.6

### ISSUE THREAD

**Issue by bespokebob** [patriksimek/vm2#399]
I haven't figured out what's going on with this change, but it's odd, and it's making some of my tests fail.

```javascript
const { NodeVM } = require('vm2')

const vm = new NodeVM({})

const result = vm.run('{}', 'vm.js')

console.log(result)
```

Output on 3.9.5:
```
{}
```

Output on 3.9.6:
```
Object {}
```

**Comment by XmiliaH** [comment]
Thanks, this is an oversight from me. I set the `Symbol.toStringTag` for Objects in the Proxy which should not happen.

**Comment by XmiliaH** [comment]
Should be fixed in v3.9.7
